backend/config/passport.js

```diff
@@ -39,22 +39,85 @@ module.exports = function () {
                 }
                 else {
                   user.verifyPassword(password, function(err, isMatch){
-                	  if (err) {
-                		done(err, null)
-                	  }
-                	  if (!isMatch) {
-                		  return done(null, false, {
+                    if (err) {
+                    done(err, null)
+                    }
+                    if (!isMatch) {
+                      return done(null, false, {
                       'errors': {
                         'password': { type: 'Password is incorrect.' }
-                    	}
+                      }
                       });
-                	  }
+                    }
                     else {
-                    	      return done(null, user);
+                            return done(null, user);
                     }
                   })
                 }
             })
           }
       ));
+// use Facebook Strategy
+
+ passport.use(new FacebookStrategy({
+
+        // pull in our app id and secret from our auth.js file
+        clientID        : configAuth.facebookAuth.clientID,
+        clientSecret    : configAuth.facebookAuth.clientSecret,
+        callbackURL     : configAuth.facebookAuth.callbackURL
+
+    },
+
+    // facebook will send back the token and profile
+    function(token, refreshToken, profile, done) {
+
+        // asynchronous
```

Comment:
@Israa-Elwan 
you should name this function and document using a form like this
/**
* @function getUsers - Called on GET "/api/users" 
* @params {Object} req - Http request
* @params {Object} res - Http response
* @params {Object} next - Next middleware
* @returns {JSON} All users in Database as JSON
*/